### ISSUE TITLE
Getting low performance for 3d_fullress

### ISSUE THREAD

**Issue by sbajpai2** [MIC-DKFZ/nnUNet#259]
Hi Fabian,

I ran Task006_Lung with 3d_fullres configuration but I am getting low dice score for it.
In the paper its 72.11 while I am getting 67.82.

I ran the same way as readme file says.
All the parameters in the 3D plans pickel file are same as written in the paper.
Can you please explain why is this so?

Best,
Shivam

**Comment by FabianIsensee** [comment]
Hi,
this dataset has questionable annotation quality and the results always vary a lot between runs. I never seed my experiments. This kind of deviation is normal for this dataset.
The results in the paper were generated with exactly the same parameters as in this repo.
You can download our pretrained models for this task if you want, no need to run the training yourself.
Best,
Fabian

**Comment by sbajpai2** [comment]
Hi Fabian,

I keep coming up with a lot of questions. I am extremely sorry for this.

Just wanted to know whether your reported results are based on regular installation of apex or python only?
Do you think that the installation procedure of apex might affect the performance?

Also, nnUNet determines the parameters and architecture configurations based on 12 GB gpu right?

Best,
Shivam Bajpai
